packages/pwa-kit-cli/src/ssr/server/build-dev-server.test.js

```diff
@@ -130,6 +130,21 @@ describe('DevServer startup', () => {
     })
 })
 
+describe('DevServer loading page', () => {
+    test('requesting homepage would temporarily redirect to the loading page, when build is not ready', async () => {
+        const options = opts()
```

Comment:
The test looks good, but the description isn't accurate. You're not testing the part of the code that conditionally redirects – you're only testing the redirect code itself. 

That's fine, but can we get the description accurate?


```suggestion
describe('DevServer loading page', () => {
    test('should redirect to the loading screen with an HTTP 302', async () => {
        const options = opts()
```